py3status/modules/file_status.py

```diff
@@ -68,16 +101,43 @@ def file_status(self):
                 'cached_until': self.py3.CACHE_FOREVER,
             }
 
-        if exists(self.path):
-            icon = self.icon_available
+        # init data
+        file_status_data = {}
+        # expand glob from paths
+        paths = list(map(glob, self.path))
+        # merge list of paths
+        paths = sorted([files for path in self.path for files in glob(path)])
+        file_status_data['paths'] = len(paths)
+
+        # fill data, legacy stuff
+        color = self.py3.COLOR_BAD
+        if paths:
             color = self.py3.COLOR_GOOD
-        else:
-            icon = self.icon_unavailable
-            color = self.py3.COLOR_BAD
```

Comment:
There sure are lot of changes. We might as well see this all the way to finish... by removing colors, make and default `thresholds` with 0, 1. With glob and several paths implemented, we might want thresholds anyway as it would enhance this module... allowing us to print colors based on number of paths somewhere.

This module is useful now because I can use this to print names of custom modules... and globbing. Thank you for working on this. :-)